tools/build/android/webrtc-native/build.gradle

```diff
@@ -66,18 +66,27 @@ dependencies {
 
 task copyToUnitySample {
     copy {
-        from './build/outputs/aar'
+        from './build/outputs/aar/merged'
         include "*.aar"
         into '../../../../libs/unity/library/Runtime/Plugins/arm64-v8a'
-    }    
+    }
+}
+
+task mergeLibwebrtcArchive(type: Exec) {
+   workingDir "$projectDir"
+   commandLine 'bash','./merge-archives.sh','-m',"$projectDir/build/outputs/aar/mrwebrtc.aar",'-o','./build/outputs/aar/merged/mrwebrtc.aar'
 }
 
 tasks.whenTaskAdded { task ->
-    if (task.name == 'externalNativeBuildRelease') {
+    // Once build is done, merge the AAR archives
+    if (task.name == 'bundleReleaseAar') {
         task.finalizedBy copyToUnitySample
-    } else if (task.name == 'externalNativeBuildDebug') {
+        copyToUnitySample.dependsOn(mergeLibwebrtcArchive)
+    } else if (task.name == 'bundleDebugAar') {
         task.finalizedBy copyToUnitySample
-    } else if (task.name == 'externalNativeBuildFulldebug') {
+        copyToUnitySample.dependsOn(mergeLibwebrtcArchive)
+    } else if (task.name == 'bundleFulldebugAar') {
         task.finalizedBy copyToUnitySample
+        copyToUnitySample.dependsOn(mergeLibwebrtcArchive)
```

Comment:
I don't know the gradle syntax, but it seems to me that the body of all 3 branches is the same 😄 Merge?

Reply:
Yeah I was lazy, didn't want to figure out Gradle. I guess some `task.name.beginsWith("externalNativeBuild")` would be best, if that's possible. I'll have a look at the Gradle docs.

Reply:
Hum.. actually I was looking at the old code. I will just add a single `if`.

Reply:
Fixed.